runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -4504,9 +4504,25 @@ TR::Register *J9::X86::TreeEvaluator::longBitCount(TR::Node *node, TR::CodeGener
 // ----------------------------------------------------------------------------
 inline void generateLoadJ9Class(TR::Node* node, TR::Register* j9class, TR::Register* object, TR::CodeGenerator* cg)
    {
+   bool needsNULLCHK = false;
+   switch (node->getOpCodeValue())
+      {
+      case TR::checkcast:
+         break;
+      case TR::checkcastAndNULLCHK:
+         needsNULLCHK = true;
+         break;
+      case TR::instanceof:
+         break;
```

Comment:
nitpicking. Can we group `TR::instanceof` with `TR::checkcast` since they do the same thing?

Reply:
Sure, updated.